.yamato/shadergraph-android-opengles3.yml

```diff
@@ -36,6 +36,9 @@ ShaderGraph_Android_OpenGLES3_Standalone_il2cpp_Linear_trunk:
                 utr --architecture=ARM64 --artifacts_path=TestProjects/ShaderGraph/test-results --platform=Android --player-load-path=players --suite=playmode --timeout=1200 --zero-tests-are-ok=1
       -  command: start %ANDROID_SDK_ROOT%\platform-tools\adb.exe kill-server
          retries: 2
+    after:
+      -  command: pip install requests --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
```

Comment:
The pip install command here is different from the one used in above jobs (`--index-url` and `--upgrade` arguments). Is that intended?

Reply:
Indeed, in some cases the pip index-url needed pointing to the specific one whereas the others worked without. But good catch, I'll add it to the others as well.